### PR TITLE
hotfix: format a fileName

### PR DESCRIPTION
**Pull request by yonatandt**
-  FileNames will not have undescore ( _ ) to avoid elastic errors.

Signed-off-by: YonatanTal <yonatald@gmail.com>

### CODE REVIEW

**Review by Shahar-Y — DISMISSED** [review]
Have you tested it in K8S?

File: service/elasticsearch/controller.go [hunk 1]
```diff
@@ -72,10 +74,25 @@ func (c Controller) Update(ctx context.Context, req *pb.File) (*pb.UpdateRespons
...
+
+	res, err := c.store.Update(ctx, file)
 	if err != nil {
 		return nil, err
 	}
 
 	return &pb.UpdateResponse{Id: res}, nil
 }
+
+// formatFile formats a givven file so there won't be elastic indexing errors.
```
Comment (nogahaskal) [inline service/elasticsearch/controller.go]: *given

**Review by nogahaskal — DISMISSED** [review]

File: service/elasticsearch/controller.go (hunk 1 above)
Comment (yonatandt) [inline service/elasticsearch/controller.go]:
```suggestion
// formatFile formats a given file so there won't be elastic indexing errors.
```

File: service/elasticsearch/controller.go [hunk 2]
```diff
@@ -72,10 +74,25 @@ func (c Controller) Update(ctx context.Context, req *pb.File) (*pb.UpdateRespons
...
+func formatFile(file *pb.File) *pb.File {
+	fileName := formatFileName(file.GetName())
+	file.Name = fileName
+	return file
+}
+
+// formatFileName formats a givven fileName to prevent elasticsearch startoffset errors.
```
Comment (yonatandt) [inline service/elasticsearch/controller.go]:
```suggestion
// formatFileName formats a given fileName to prevent elasticsearch startoffset errors.
```

**Pull request merged**